File: back-end/Models/Lote/Lote.cs
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;

namespace arBrain.Models.Lote
{
    public class Lote
    {
        public int LoteId { get; set; }

        [MaxLength(200)]
        public string LoteDescricao { get; set; } = string.Empty;

        [Precision(10, 2)]
        public decimal LoteQuantidade { get; set; }

        [MaxLength(2000)]
        public string? LoteObservacao { get; set; }

        public int TanqueId { get; set; }

        public int CervejaId { get; set; }

        public DateTime LoteInicio { get; set; }

        public DateTime? LoteFinalizacao { get; set; }
    }
}

File: frontend/src/models/Lote.ts
export interface Lote {
  loteId          : number;
  loteDescricao   : string;
  loteQuantidade  : number;
  loteObservacao  : string | null;
  tanqueId        : number;
  cervejaId       : number;
  loteInicio      : string;
  loteFinalizacao : string | null;
}